pkgroot/usr/local/outset/outset

```diff
@@ -420,14 +420,16 @@ def main():
 
     if args.login:
         if console_user not in ignored_users:
-            if os.listdir(login_once_dir):
-                process_items(login_once_dir, once=True, override=override_login_once)
-            if os.listdir(login_every_dir):
-                process_items(login_every_dir)
-            if os.listdir(login_privileged_once_dir) or os.listdir(login_privileged_every_dir):
-                open(login_privileged_trigger, 'a').close()
-        else:
-            logging.info('Skipping login scripts for user %s', console_user)
+			if os.listdir(login_once_dir):
+				process_items(login_once_dir, once=True, override=override_login_once)
+			current_user = os.environ['USER']
+			if console_user == current_user:
```

Comment:
I appreciate this fixes your issue with profiles, but this looks like this could introduce issues with other things. Why would you skip other things when `console_user != os.environ['USER']` ?

Reply:
Seems to me that a safer fix is to add this check (`current_user == os.environ['USER']`) only for per-user profile installs